src/transformers/trainer.py

```diff
@@ -1096,7 +1096,35 @@ def call_model_init(self, trial=None):
 
         return model
 
-    def _wrap_model(self, model, training=True):
+    def torch_jit_model(self, model, training=False, dataloader=None):
+        if not training:
+            jit_inputs = ()
+            for _, batch in enumerate(dataloader):
+                for _, label in enumerate(batch):
+                    dumpy_tensor = torch.ones_like(batch[label])
+                    L1 = list(jit_inputs)
+                    L1.append(dumpy_tensor)
```

Comment:
Please use a better variable name that is more descriptive of what `L1` is.

Reply:
`L1 `was a temp list, here made changes to just use `jit_inputs` list  to avoid confusing, thanks.